tests/snuba/sessions/test_sessions.py

```diff
@@ -1779,6 +1789,10 @@ def test_get_current_and_previous_crash_free_rates_with_zero_sessions(self):
         }
 
 
+class GetCrashFreeRateTestCaseMetrics(ReleaseHealthMetricsTestCase, GetCrashFreeRateTestCase):
+    """Repeat tests with metrics backend"""
+
```

Comment:
Please Add a `ToDo` comment

Reply:
Deriving from both `ReleaseHealthMetricsTestCase` and `GetCrashFreeRateTestCase` already repeats the tests, so there is nothing to do here. But I agree that that should be clearer from reading the code.